test/AspNetCoreModule.TestSites.Standard/Startup.cs

```diff
@@ -318,6 +318,31 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
                             response += de.Key + ":" + de.Value + "<br/>";
                         }
                     }
+
```

Comment:
you can use query string in url for better test maintenance. You can do it later